packages/site/src/data/graphql/libraries.ts

```diff
@@ -218,7 +238,7 @@ export const libraries: Library<typeof libraryTags[number]>[] = [
 			"GraphQL Code Generator is a tool that generates code out of your GraphQL schema.",
 		image:
 			"https://raw.githubusercontent.com/dotansimha/graphql-code-generator/master/logo.svg",
-		tags: ["Schema"],
+		tags: ["Schema", "JavaScript", "NodeJS", "Java"],
```

Comment:
I guess this isn't related to Java, right?

Reply:
I removed it 👍